src/components/app.test.ui.tsx

```diff
@@ -133,7 +133,7 @@ describe('Voyager', () => {
               const fieldList = wrapper.find('.encoding-shelf__encoding-shelf');
               const fields = fieldList.map(d => d.text());
 
-              expect(fields).toContain('x close');
+              expect(fields).toContain('x binclose');
```

Comment:
the spec is binned, and this text is from the UI with bin(close)  but parenthesis are displayed using before: and after: element so they are not included here.